### PR TITLE
Add the set of currently used patches

### PR DESCRIPTION
**Pull request by danimo**

### CODE REVIEW

File: admin/qt/patches/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Patches used
+
+There are our patches on top of Qt 5.4.0, which we are currently
```
Comment (phil-davis) [inline admin/qt/patches/README.md]: s/There/These/

File: admin/qt/patches/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+using for our binary packages on Windows and Mac OS. Most of them
+have been sent upstream and are part of newer Qt releases.
+
+All changes are designed to up upstream, and all those that are
```
Comment (phil-davis) [inline admin/qt/patches/README.md]: to (go?) upstream

**Comment by dragotin** [comment]
:+1: nice.

**Comment by dragotin** [comment]
Merged into branch 2.1. Closing this one.

**Pull request closed without merging**